apps/condo/domains/user/access/RegisterUserExternalIdentityService.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Generated by `createservice user.RegisterUserExternalIdentityService`
+ */
+const access = require('@open-condo/keystone/access')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canRegisterUserExternalIdentity (args) {
+    const { args: { data: { user: { id: userId } } }, authentication: { item: user } } = args
+
+    if (!access.userIsAuthenticated(args)) {
+        return false
+    }
+
+    return user.type === RESIDENT && user.id === userId
```

Comment:
It looks strant why we really give access to the user to link to themself any external identity IDs ?!

Reply:
Good point. I'll remove that permission